bundles/org.openhab.binding.boschshc/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -17,19 +17,19 @@
                 <required>true</required>
             </parameter>
             <parameter name="password" type="text">
-                <label>Network Address</label>
-                <description>Network address of the Bosch Smart Home Controller.</description>
+                <label>System password</label>
+                <description>The system password of the Bosch Smart Home Controller necessary for pairing.</description>
                 <required>true</required>
             </parameter>
             <parameter name="keystorePath" type="text">
                 <label>Java KeyStore path.</label>
-                <description>Path at which the keystore with SHC access keys is stored.</description>
+                <description>Path at to a *.jks keystore file with Self Signed Certificate. If it doesn't exist it will created.</description>
```

Comment:
Would it be possible to make this path optional and use a default path to store the keystore at? As an average user I wouldn't have a clue what to enter here :)

Reply:
Just saw the TODO in the BoschSHCBridgeHandler.java about finding a default path. So, yes, that would be nice!

Reply:
The only place that seems to work for this kind of data seems to be `openhab\userdata\etc` At least in my Docker openHAB installation. Maybe we can create a sub-folder `boschshc` there?

Reply:
That's something I'd hope to learn once we open a PR against master. The devs there should be able to tell us where we should store it :smile:

Reply:
If there is an API or enviroment variable available to get the usedata folder I will use it and we skip the two additional JKS related settings. Sometimes the IP address and the system password can be a problem during setup.